### DISCUSSION TITLE
on_event('shutdown') not work in windows

### DISCUSSION THREAD

**Discussion by ljd1995** [fastapi/fastapi#6026] [Questions] [answered]
### First Check

- [X] I added a very descriptive title here.
- [X] I used the GitHub search to find a similar question and didn't find it.
- [X] I searched the FastAPI documentation, with the integrated search.
- [X] I already searched in Google "How to X in FastAPI" and didn't find any information.
- [X] I already read and followed all the tutorial in the docs and didn't find an answer.
- [X] I already checked if it is not related to FastAPI but to [Pydantic](https://github.com/pydantic/pydantic).
- [X] I already checked if it is not related to FastAPI but to [Swagger UI](https://github.com/swagger-api/swagger-ui).
- [X] I already checked if it is not related to FastAPI but to [ReDoc](https://github.com/Redocly/redoc).

### Commit to Help

- [X] I commit to help with one of those options 👆

### Example Code

```python
from fastapi import FastAPI

def register_init(app: FastAPI) -> None:
    @app.on_event("startup")
    async def init_connect():
        print("init_connect...")

    @app.on_event('shutdown')
    async def shutdown_connect():
        print("shutdown_connect...")


def create_app() -> FastAPI:
    app = FastAPI()
    
    register_init(app) 
    return app


app = create_app()

if __name__ == "__main__":
    import uvicorn
    uvicorn.run(app='main:app', host="127.0.0.1", port=8010, reload=True)
```

### Description

I tested this code and it works fine in Linux., but Windows does not output, please check Windows platform compatibility
Windows：
![image](https://user-images.githubusercontent.com/17735771/220332906-417d7e62-42c4-49e0-b5e1-ba917e7eb746.png)
Linux：
![image](https://user-images.githubusercontent.com/17735771/220332997-f0a18817-e7e3-424f-928f-86082fc0b162.png)

### Operating System

Windows

### FastAPI Version

0.90.0

### Python Version

3.9

**Comment by maxwelldorliea** [comment]
Windows what? Which build number?

**Reply by ljd1995** [reply]
> Windows what? Which build number?

windows11    22H2     22621.1265

**Reply by mr-st0rm** [reply]
it works on windows 10

**Comment by fliiiix** [comment, marked as answer]
Same issue as https://github.com/tiangolo/fastapi/discussions/6391 ?
